lib/page.rb

```diff
@@ -77,6 +77,20 @@ def self.execute_script(script)
       Driver.execute_script_driver(script)
     end
 
+    def self.evaluate_script(script)
```

Comment:
> We have this functionality available in the driver object.

I was going with the convention already implemented [here](https://github.com/rizzza/gridium/blob/a9d193ef8add83d8d1946012277bd54e7b96897f/lib/page.rb#L76-L78)

To me, it makes sense to have it available in the `Page` object encapsulating the `Driver`.